src/Microsoft.Win32.Registry/src/Microsoft.Win32.Registry.csproj

```diff
@@ -6,16 +6,11 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <NoWarn>$(NoWarn);CS1573</NoWarn>
     <DefineConstants>$(DefineConstants);REGISTRY_ASSEMBLY</DefineConstants>
-    <IsPartialFacadeAssembly Condition="'$(TargetsNetFx)' == 'true'">true</IsPartialFacadeAssembly>
-    <GeneratePlatformNotSupportedAssemblyMessage Condition="'$(OSGroup)' == 'AnyOS' AND '$(TargetGroup)' == 'netstandard'">SR.PlatformNotSupported_Registry</GeneratePlatformNotSupportedAssemblyMessage>
-    <!-- 
-        Temporarily necessary as this is only being built for UWP due to an unwanted dependency on it 
-        https://github.com/dotnet/corefx/issues/15966
-      -->
     <EnablePinvokeUWPAnalyzer>false</EnablePinvokeUWPAnalyzer>
```

Comment:
Can you remove the property?

Reply:
Removing that property triggers the following errors:

```

D:\a\1\s\src\Common\src\CoreLib\Interop\Windows\Advapi32\Interop.RegCloseKey.cs(12,9): error BCL0015: advapi32.dll!RegCloseKey is not supported on one\more targeted platforms. [D:\a\1\s\src\Microsoft.Win32.Registry\src\Microsoft.Win32.Registry.csproj]
D:\a\1\s\src\Common\src\Interop\Windows\Advapi32\Interop.RegConnectRegistry.cs(18,9): error BCL0015: advapi32.dll!RegConnectRegistryW is not supported on one\more targeted platforms. [D:\a\1\s\src\Microsoft.Win32.Registry\src\Microsoft.Win32.Registry.csproj]
D:\a\1\s\src\Common\src\CoreLib\Interop\Windows\Advapi32\Interop.RegCreateKeyEx.cs(20,9): error BCL0015: advapi32.dll!RegCreateKeyExW is not supported on one\more targeted platforms. [D:\a\1\s\src\Microsoft.Win32.Registry\src\Microsoft.Win32.Registry.csproj]
D:\a\1\s\src\Common\src\CoreLib\Interop\Windows\Advapi32\Interop.RegDeleteKeyEx.cs(17,9): error BCL0015: advapi32.dll!RegDeleteKeyExW is not supported on one\more targeted platforms. [D:\a\1\s\src\Microsoft.Win32.Registry\src\Microsoft.Win32.Registry.csproj]
D:\a\1\s\src\Common\src\CoreLib\Interop\Windows\Advapi32\Interop.RegDeleteValue.cs(18,9): error BCL0015: advapi32.dll!RegDeleteValueW is not supported on one\more targeted platforms. [D:\a\1\s\src\Microsoft.Win32.Registry\src\Microsoft.Win32.Registry.csproj]
D:\a\1\s\src\Common\src\CoreLib\Interop\Windows\Advapi32\Interop.RegEnumKeyEx.cs(19,9): error BCL0015: advapi32.dll!RegEnumKeyExW is not supported on one\more targeted platforms. [D:\a\1\s\src\Microsoft.Win32.Registry\src\Microsoft.Win32.Registry.csproj]
D:\a\1\s\src\Common\src\CoreLib\Interop\Windows\Advapi32\Interop.RegEnumValue.cs(18,9): error BCL0015: advapi32.dll!RegEnumValueW is not supported on one\more targeted platforms. [D:\a\1\s\src\Microsoft.Win32.Registry\src\Microsoft.Win32.Registry.csproj]
D:\a\1\s\src\Common\src\CoreLib\Interop\Windows\Advapi32\Interop.RegFlushKey.cs(16,9): error BCL0015: advapi32.dll!RegFlushKey is not supported on one\more targeted platforms. [D:\a\1\s\src\Microsoft.Win32.Registry\src\Microsoft.Win32.Registry.csproj]
D:\a\1\s\src\Common\src\CoreLib\Interop\Windows\Advapi32\Interop.RegOpenKeyEx.cs(18,9): error BCL0015: advapi32.dll!RegOpenKeyExW is not supported on one\more targeted platforms. [D:\a\1\s\src\Microsoft.Win32.Registry\src\Microsoft.Win32.Registry.csproj]
D:\a\1\s\src\Common\src\CoreLib\Interop\Windows\Advapi32\Interop.RegOpenKeyEx.cs(27,9): error BCL0015: advapi32.dll!RegOpenKeyExW is not supported on one\more targeted platforms. [D:\a\1\s\src\Microsoft.Win32.Registry\src\Microsoft.Win32.Registry.csproj]
D:\a\1\s\src\Common\src\CoreLib\Interop\Windows\Advapi32\Interop.RegQueryInfoKey.cs(19,9): error BCL0015: advapi32.dll!RegQueryInfoKeyW is not supported on one\more targeted platforms. [D:\a\1\s\src\Microsoft.Win32.Registry\src\Microsoft.Win32.Registry.csproj]
D:\a\1\s\src\Common\src\CoreLib\Interop\Windows\Advapi32\Interop.RegQueryValueEx.cs(19,9): error BCL0015: advapi32.dll!RegQueryValueExW is not supported on one\more targeted platforms. [D:\a\1\s\src\Microsoft.Win32.Registry\src\Microsoft.Win32.Registry.csproj]
D:\a\1\s\src\Common\src\CoreLib\Interop\Windows\Advapi32\Interop.RegQueryValueEx.cs(28,9): error BCL0015: advapi32.dll!RegQueryValueExW is not supported on one\more targeted platforms. [D:\a\1\s\src\Microsoft.Win32.Registry\src\Microsoft.Win32.Registry.csproj]
D:\a\1\s\src\Common\src\CoreLib\Interop\Windows\Advapi32\Interop.RegQueryValueEx.cs(37,9): error BCL0015: advapi32.dll!RegQueryValueExW is not supported on one\more targeted platforms. [D:\a\1\s\src\Microsoft.Win32.Registry\src\Microsoft.Win32.Registry.csproj]
D:\a\1\s\src\Common\src\CoreLib\Interop\Windows\Advapi32\Interop.RegQueryValueEx.cs(46,9): error BCL0015: advapi32.dll!RegQueryValueExW is not supported on one\more targeted platforms. [D:\a\1\s\src\Microsoft.Win32.Registry\src\Microsoft.Win32.Registry.csproj]
D:\a\1\s\src\Common\src\CoreLib\Interop\Windows\Advapi32\Interop.RegSetValueEx.cs(18,9): error BCL0015: advapi32.dll!RegSetValueExW is not supported on one\more targeted platforms. [D:\a\1\s\src\Microsoft.Win32.Registry\src\Microsoft.Win32.Registry.csproj]
D:\a\1\s\src\Common\src\CoreLib\Interop\Windows\Advapi32\Interop.RegSetValueEx.cs(27,9): error BCL0015: advapi32.dll!RegSetValueExW is not supported on one\more targeted platforms. [D:\a\1\s\src\Microsoft.Win32.Registry\src\Microsoft.Win32.Registry.csproj]
D:\a\1\s\src\Common\src\CoreLib\Interop\Windows\Advapi32\Interop.RegSetValueEx.cs(36,9): error BCL0015: advapi32.dll!RegSetValueExW is not supported on one\more targeted platforms. [D:\a\1\s\src\Microsoft.Win32.Registry\src\Microsoft.Win32.Registry.csproj]
D:\a\1\s\src\Common\src\CoreLib\Interop\Windows\Advapi32\Interop.RegSetValueEx.cs(45,9): error BCL0015: advapi32.dll!RegSetValueExW is not supported on one\more targeted platforms. [D:\a\1\s\src\Microsoft.Win32.Registry\src\Microsoft.Win32.Registry.csproj]
D:\a\1\s\src\Common\src\CoreLib\Interop\Windows\Advapi32\Interop.RegSetValueEx.cs(54,9): error BCL0015: advapi32.dll!RegSetValueExW is not supported on one\more targeted platforms. [D:\a\1\s\src\Microsoft.Win32.Registry\src\Microsoft.Win32.Registry.csproj]
```

Reply:
I see you referenced here: https://github.com/dotnet/corefx/issues/15966#issuecomment-475210513

I wonder if we even care about this analyzer in its current form any more.  Is AppContainer / WACK even a constraint now that phones are gone and Windows is being more flexible about what APIs apps use?  Anyhow, I'm OK with you keeping the attribute, I was mainly concerned that you were deleting the issue breadcrumb without acting on it.

Reply:
> I wonder if we even care about this analyzer in its current form any more. Is AppContainer / WACK even a constraint now that phones are gone and Windows is being more flexible about what APIs apps use?

I agree with what you are saying but I believe we shouldn't make changes to it at this point. As soon as there is a decision made regarding uap support in corefx we can take action.

> Anyhow, I'm OK with you keeping the attribute, I was mainly concerned that you were deleting the issue breadcrumb without acting on it.

I see. Maybe I should leave it there.

Reply:
I'm fine either way now that I understand what's going on.  You mentioned in that issue that it was intentional to leave the property so you can delete the comment.  We should just have an issue deciding repo-wide what we do with the UWP PInvoke analyzer.  I'm pretty sure folks are largely ignoring it and either disabling it or base-lining errors today.

Reply:
OK great. I'm sure the mentioned decision will come sooner or later which will impact our entire configuration matrix similar to what we did with uapaot recently.